### PR TITLE
Update russian translation of Quartz add-on

### PR DESCRIPTION
**Pull request by daring2**
Original PR - #1329

Changes:
- add translation for add-on entities, properties, enums.
- change JobModel entity caption from "Задача" to "Задание".
- fix some typos (for example, `jobDataParamKeyIsRequired`, `jobDataParamKeyAlreadyExistsValidationMessage` messages).

### CODE REVIEW

File: jmix-translations/content/io/jmix/quartz/messages_ru.properties [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+io.jmix.quartz.model/JobModel.jobSource=Источник
+io.jmix.quartz.model/JobModel.description=Описание
+
+io.jmix.quartz.model/JobSource=Источник задания
+io.jmix.quartz.model/JobSource.USER_DEFINED=Пользовательское
+io.jmix.quartz.model/JobSource.PREDEFINED=Предопределенное
+
+io.jmix.quartz.model/JobState=Состояние задания
+io.jmix.quartz.model/JobState.NORMAL=Активное
+io.jmix.quartz.model/JobState.PAUSED=Приостановлено
```
Comment (Gavrilov-Ivan) [inline jmix-translations/content/io/jmix/quartz/messages_ru.properties:39]: There should be the same form for both states:
'Активно' - 'Приостановлено' or 'Активное' - 'Приостановленное'.
The first one is a little better, I think.

File: jmix-translations/content/io/jmix/quartzui/messages_ru.properties [hunk 2]
```diff
@@ -15,52 +15,53 @@
...
-                    <li>0 0 0 ? * * - срабатывает в 00:00 каждый день</li>\
-                    <li>0 0 8-10 ? * * * - срабатывает в 8, 9 и 10 часов каждый день</li>\
-                    <li>0 0/30 8-10 ? * * * - срабатывает каждый день в 8:00, 8:30, 9:00, 9:30 и 10 часов</li>\
-                    <li>0 10,44 14 ? 3 WED - срабатывает в 14:10 и в 14:44 каждую среду в марте</li>\
-                    <li>0 15 10 ? * 6#3 - срабатывает в 10:15 в третью пятницу каждого месяца</li>\
-                    <li>0 11 11 11 11 ? - срабатывает каждое 11 ноября в 11:11</li>\
-                    <li>0 0 0 7 1 ? - запускается каждое Рождество в полночь</li>\
+                    <li>0 0/5 * ? * * - запускать каждые 5 минут </li>\
+                    <li>0 0 0 ? * * - запускать каждый день</li>\
+                    <li>0 0 8-10 * * * - запускать каждый день в 8:00, 9:00 and 10:00</li>\
```
Comment (Gavrilov-Ivan) [inline jmix-translations/content/io/jmix/quartzui/messages_ru.properties:47]: and -> и

File: jmix-translations/content/io/jmix/quartzui/messages_ru.properties [hunk 3]
```diff
@@ -15,52 +15,53 @@
...
+                    <li>0 0/30 8-10 * * * - запускать каждый день в 8:00, 8:30, 9:00, 9:30 and 10:00</li>\
```
Comment (Gavrilov-Ivan) [inline jmix-translations/content/io/jmix/quartzui/messages_ru.properties:48]: and -> и

File: jmix-translations/content/io/jmix/quartzui/messages_ru.properties [hunk 4]
```diff
@@ -15,52 +15,53 @@
...
 io.jmix.quartzui.screen.trigger/RepeatMode.EXECUTE_FOREVER=Бесконечные запуски
 io.jmix.quartzui.screen.trigger/triggerRepeatMode=Режим повтора
-io.jmix.quartzui.screen.trigger/triggerRepeatModeHelpText=Режим повтора определяет количество запусков триггера:\
+io.jmix.quartzui.screen.trigger/triggerRepeatModeHelpText=Режим повтора определяет количество запусков задания:\
                 <ul>\
-                    <li>Однократный запуск - триггер запустится только один раз</li>\
-                    <li>Ограниченное количество запусков - триггер запустится 'Repeat count' + 1 (изначальный) раз каждые 'Repeat interval' мс</li>\
-                    <li>Бесконечные запуски - триггер будет запускаться бесконечно каждые 'Repeat interval' мс</li>\
+                    <li>Однократный запуск - задание запустится только один раз</li>\
+                    <li>Ограниченное количество запусков - задание запустится 'Количество повторений' + 1 (первый запуск)\
```
Comment (Gavrilov-Ivan) [inline jmix-translations/content/io/jmix/quartzui/messages_ru.properties:61]: Missed 'раз'

**Review by Gavrilov-Ivan — CHANGES_REQUESTED** [review]
Hello.
Added some comments.
And please add the same changes (from quartzui) to quartzflowui package.

File: jmix-translations/content/io/jmix/quartz/messages_ru.properties (hunk 1 above)
Comment (daring2) [reply jmix-translations/content/io/jmix/quartz/messages_ru.properties:39]: Fixed

**Comment by daring2** [comment]
I fixed typos and updated quartzflowui messages.

**Review by Gavrilov-Ivan — APPROVED** [review]

**Pull request merged**